command/app.go

```diff
@@ -81,6 +81,15 @@ var app = &cli.App{
 			Name:  "request-payer",
 			Usage: "who pays for request (access requester pays buckets)",
 		},
+		&cli.StringFlag{
+			Name:  "profile",
+			Usage: "use the specified profile from the credentials file",
+		},
+		&cli.StringFlag{
+			Name:    "credentials-file",
+			Aliases: []string{"cf"},
```

Comment:
I don't think we need an alias for this flag :)

Reply:
Flag name is so long so I think it would be nice to have a short alias. But since it is not supposed to be a common flag, I agree it's a bit unnecessary.

I'll remove. Thanks!